packages/block-editor/src/store/defaults.js

```diff
@@ -131,6 +132,12 @@ export const SETTINGS_DEFAULTS = {
 		{ slug: 'full', name: __( 'Full Size' ) },
 	],
 
+	padding: [
```

Comment:
cc @mapk @kjellr @jasmussen I'm still working on this one, but I thought I'd ping early. What do you think of these default values? Would you use a different combination?

Reply:
I'd go in multiples of 4 if we go with pixels or even another unit. I'm wondering if rem units might be better here.

As far as pixel values
```
small: 4 - nice
regular: 16
large: 32 - only large relative to body copy, though
```

Em units could also work well and more flexibly even if they are harder to understand. If so, maybe:
```
small: .5em
regular: 1em
large: 1.5em
```

Reply:
Thanks, Michael! I pushed those defaults. I also wanted to say that this PR is going to hibernate for a bit as there are some design and engineering issues we need to fix before being able to land it (so it's not going to be part of 5.5 -- sorry about the confusion!).